nautobot/core/utilities.py

```diff
@@ -1,5 +1,6 @@
 from django.core.exceptions import FieldError
 
+from nautobot.extras.filters.mixins import RelationshipFilter
```

Comment:
I don't *like* importing from `nautobot.extras` into `nautobot.core` but I guess if it's not introducing a circular dependency it's OK for now.